### DISCUSSION TITLE
Cannot use `!important` with `@apply`

### DISCUSSION THREAD

**Discussion by konsalex** [tailwindlabs/tailwindcss#10799] [Help]
hey folks, trying to use `!important` with apply, and as many links suggest, or the docs I should use `#{!important}`, but when I do I get an error:

```bash
CssSyntaxError: postcss-import: cypher-code-block.pcss:139:22: Unknown word
```

Any clue why? I suspect plugin issue? My `postcss.config.js` looks like:

```js
module.exports = {
  plugins: {
    'postcss-import': {},
    'tailwindcss/nesting': {},
    tailwindcss: {},
    autoprefixer: {},
  },
};
```

**Comment by wongjn** [comment]
I believe the `#{!important}` syntax is for when using Sass to work-around a parsing issue. In any case, you could use the [important modifier](https://tailwindcss.com/docs/configuration#important-modifier) instead, like `@apply !text-black`.

**Reply by konsalex** [reply]
I think this works for inline classes @wongjn , tried this inside my CSS file and I am getting some errors `CssSyntaxError: cypher-code-block.pcss:139:5: The `!ndl-btn` class does not exist. If `!ndl-btn` is a custom class, make sure it is defined within a `@layer` directive.`

**Reply by wongjn** [reply]
It seems like the `ndl-btn` class is not known to Tailwind. Ensure it is either derived from a Tailwind plugin or its rule is defined in a `@layer`. [Here's a working example](https://play.tailwindcss.com/l6qJeLNgQo?file=css) (open the *Generated CSS* panel to see the compiled CSS).

**Reply by konsalex** [reply]
It is, as it works witouth the `!important` in front as expected

**Reply by wongjn** [reply]
Consider providing a remotely-hosted git repository that reproduces the issue.
